calliope/test/test_constraint_results.py

```diff
@@ -8,7 +8,7 @@
 
 
 class TestNationalScaleExampleModelSenseChecks:
```

Comment:
Again, these are essentially duplicating tests now given by the minimal group constraint models, do we need them?

Reply:
These tests actually picked up issues with the group constraints that the group constraint tests did not, so they are not duplicating them (again, the group constraints need better tests before 0.6.4 release)

Reply:
Yeah, I'd prefer that the group constraint tests are kept to their respective minimal examples, and that we don't have to pick up strange edge cases using the national-scale / urban-scale example models... If nothing else, it should keep test time down

Reply:
We should consider this when finishing up #193